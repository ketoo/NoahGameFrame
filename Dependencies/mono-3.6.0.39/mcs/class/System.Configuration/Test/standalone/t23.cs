using System;
using System.Configuration;

class T1
{
	static void Main(string[] args)
	{
		try {
			Console.WriteLine ("DefaultProvider = {0}", ProtectedConfiguration.DefaultProvider);
			foreach (ProtectedConfigurationProvider pc in ProtectedConfiguration.Providers) {
				Console.WriteLine (pc.Name);
				if (pc is RsaProtectedConfigurationProvider) {
					RsaProtectedConfigurationProvider rsa = (RsaProtectedConfigurationProvider)pc;

					Console.WriteLine ("keyContainerName = {0}", rsa.KeyContainerName);
					Console.WriteLine ("useMachineContainer = {0}", rsa.UseMachineContainer);
				}
			}
		}
		catch (Exception e)
		{
			Console.WriteLine ("Exception raised: {0}", e.Message);
		}
	}
}
