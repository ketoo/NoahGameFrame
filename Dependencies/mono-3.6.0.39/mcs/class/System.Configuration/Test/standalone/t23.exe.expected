DefaultProvider = MyRsaProtectedConfigurationProvider
Exception raised: The type specified does not extend ProtectedConfigurationProvider class.
